### PR TITLE
fix index arithmetic in entropy calculation

### PR DESCRIPTION
**Pull request by dodgejoel**

### CODE REVIEW

File: _mdlp.pyx [hunk 1]
```diff
@@ -96,7 +96,7 @@ cdef bint reject_split(np.ndarray[np.int64_t, ndim=1] y, int start, int end, int
 
     # Calculate the final values
     cdef:
-        float part1 = 1 / N * ((start - k) * entropy1 + (end - k) * entropy2)
+        float part1 = 1 / N * ((k - start) * entropy1 + (end - k) * entropy2)
```
Comment (hlin117) [inline _mdlp.pyx:99]: Ahh you're right, my bug would have indeed created a negative value in the calculations...

I would like this to be regression tested. But I can't do that until we fix the unit tests in this package, as well as the Travis build.

Comment (hlin117) [reply _mdlp.pyx:99]: As part of this PR, do you mind getting the unit tests to work in this package?

Comment (dodgejoel) [reply _mdlp.pyx:99]: Sure. I have reworked the testing setup instructions and added a requirements-test.txt file. You were not clear about exactly what was wrong with your runs of the unit tests but they pass for me on the next commit. 

re travis build. it looks like the configuration may be messed up or something. I have never interacted with travis before but it seems odd that there would be references to ruby in the config. Maybe sort that out?

Comment (dodgejoel) [reply _mdlp.pyx:99]: I fooled around with travis a bit and indeed it seems that your first problem was not having a config file i place. I added one on my branch. Now the problem is some binary mismatch between the version of numpy that travis is using in builds and the version of numpy that (i think) sklearn was compiled with.

**Comment by dodgejoel** [comment]
ill submit another pull request with a fix for the travis build

**Pull request closed without merging**